### DISCUSSION TITLE
Any live websites using Openclassify?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [openclassify/openclassify#1367] [General]
Hi guys, do you know any live websites using Openclassify? Is the project dead??

**Comment by tradenet** [comment]
Would be interesting to know.

**Comment by agenthinson99** [comment]
We offer all types of financial loans with an interest rate of 2%. We offer a wide range of financial services which includes: Business planning, commercial and development finance, properties and mortgages, debt consolidation loans, business loans, private loans, home refinancing, hotel loans, student loans etc. Kindly respond immediately via email below for more information.

Lender's Email: agenthinson@gmail.com

Interested applicants should apply with the following information below:-

Name:
Country:
Loan Amount: 
Duration:
WhatsApp Phone#:
Gender:
![Uploading aa.jpg…]()

**Reply by agenthinson99** [reply]
We offer all types of financial loans with an interest rate of 2%. We offer a wide range of financial services which includes: Business planning, commercial and development finance, properties and mortgages, debt consolidation loans, business loans, private loans, home refinancing, hotel loans, student loans etc. Kindly respond immediately via email below for more information.

Lender's Email: agenthinson@gmail.com

Interested applicants should apply with the following information below:-

Name:
Country:
Loan Amount: 
Duration:
WhatsApp Phone#:
Gender:
![Uploading aa.jpg…]()

**Comment by gijoe707** [comment]
https://emlaktura.com/
@vitalijalbu @tradenet
